shared_libraries/API.md

```diff
@@ -50,7 +50,7 @@ Environment Variables:
     AWS_REGION (str): AWS reserved runtime variable used to set boto3 client region.
 
 Parameter Store:
-    <prefix>-orca-db-login-secret (string): The json string containing all the admin and user db login info.
+    <prefix>-orca-db-login-secret (string): The json string containing all the db login info.
```

Comment:
`API.md` is generated from `bin/build_api.sh` These changes should be made in the codes doc strings for the proper item and the file regenerated.

Reply:
Yup. Looks like the last updates to these files didn't make it into API.md. These showed up when I ran build_api.sh